composer.json

```diff
@@ -122,7 +122,9 @@
         "phpunit/phpunit": "4.5.0",
 
         "mmoreram/php-formatter": "1.0.4",
-        "fabpot/php-cs-fixer": "1.4.2"
+        "fabpot/php-cs-fixer": "1.4.2",
+        "visithor/visithor-bundle": "0.1.*@dev",
```

Comment:
Ah! Sorry! I thought you were talking about php-cs-formatter. :'(

This is already changed :D